system/system_monitor/src/net_monitor/net_monitor.cpp

```diff
@@ -46,10 +48,15 @@ NetMonitor::NetMonitor(const rclcpp::NodeOptions & options)
   last_update_time_{0, 0, this->get_clock()->get_clock_type()},
   device_params_(
     declare_parameter<std::vector<std::string>>("devices", std::vector<std::string>())),
+  last_reassembles_failed_(0),
   monitor_program_(declare_parameter<std::string>("monitor_program", "greengrass")),
   traffic_reader_port_(declare_parameter<int>("traffic_reader_port", TRAFFIC_READER_PORT)),
   crc_error_check_duration_(declare_parameter<int>("crc_error_check_duration", 1)),
-  crc_error_count_threshold_(declare_parameter<int>("crc_error_count_threshold", 1))
+  crc_error_count_threshold_(declare_parameter<int>("crc_error_count_threshold", 1)),
+  reassembles_failed_check_duration_(
+    declare_parameter<int>("reassembles_failed_check_duration", 1)),
+  reassembles_failed_check_count_(declare_parameter<int>("reassembles_failed_check_count", 1)),
+  reassembles_failed_column_index_(-1)
```

Comment:
Better to initialize as 0. 0 implies row header.

Reply:
@ito-san 
I changed the initial value of the variable to 0.
ef247d72f3cebf0ae6cf21935dc4d86543f161f1